integreat_cms/matomo_api/apps.py

```diff
@@ -11,8 +11,9 @@
 from django.conf import settings
 from django.utils.translation import ugettext as _
 
+# pylint: disable=relative-beyond-top-level
```

Comment:
```suggestion
```
> I had to disable one pylint test. I kept failing, although as far as I understood the message I already had solved it. It keep reporting, that I should put the relative imports to the bottom. But that was already the case.

The problem is that you don't have an `__init__.py` file in the `matomo_api` directory - thus, `apps.py` is considered a standalone python module, so the "top level" in this case is `apps.py` and you cannot import relative paths outside of this file.